ssl/statem/extensions.c

```diff
@@ -121,6 +121,14 @@ static const EXTENSION_DEFINITION ext_defs[] = {
         final_renegotiate
     },
     {
+        TLSEXT_TYPE_max_fragment_length,
+        SSL_EXT_CLIENT_HELLO | SSL_EXT_TLS1_3_ENCRYPTED_EXTENSIONS
+        | SSL_EXT_TLS1_2_SERVER_HELLO | SSL_EXT_TLS1_3_SERVER_HELLO,
+        NULL, tls_parse_ctos_maxpacketlen, tls_parse_stoc_maxpacketlen,
+        tls_construct_stoc_maxpacketlen, tls_construct_ctos_maxpacketlen,
```

Comment:
Why are these functions called ...maxpacketlen and not ...maxfragmentlen? Kinda confusing.